_unused_topics/cluster-logging-exported-fields-kubernetes.2021-06-04.adoc

```diff
@@ -0,0 +1,83 @@
+[id="cluster-logging-exported-fields-kubernetes_{context}"]
+= Kubernetes
+
+The following fields may be present in the namespace for kubernetes-specific metadata.
```

Comment:
Is `may` the right word here? Isn't there something in the style guide about avoiding the subjunctive mood?

What about `The following fields are sometimes present in the......`

Reply:
Thanks for your review, Neal. 
Yes. The IBM SG says to avoid the subjunctive mood. About **may**, it says, "Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.
"May" is overused in technical information. In most cases, use "can" or "might" instead."

Although this is an unused topic, I'll make the change in case it slides back into normal use.